esx_service/vsan_policy.py

```diff
@@ -67,6 +68,7 @@ def create_policy_file(filename, content):
             f.write(content)
             f.write('\n')
     except:
+        logging.exception("Failed to open %s for writing", filename)
```

Comment:
There is one place in vmdk_ops.c: setStatusDetached() where we do a try: - except: and in the except part we do a "pass" if there is an exception trying to delete the VM name and UUID. This is wrong because meta-data is missing and should be logged.